src/main/java/org/eclipse/pathfinder/api/GraphTraversalService.java

```diff
@@ -50,53 +50,34 @@ public List<TransitPath> findShortestPath(
       @Size(min = 8, max = 8, message = "Deadline value must be eight characters long.")
           @QueryParam("deadline")
           String deadline) {
-    LocalDateTime date = nextDate(LocalDateTime.now());
 
     List<String> allVertices = dao.listLocations();
     allVertices.remove(originUnLocode);
     allVertices.remove(destinationUnLocode);
 
     int candidateCount = getRandomNumberOfCandidates();
-    List<TransitPath> candidates = new ArrayList<>(candidateCount);
+    List<TransitPath> candidates = new ArrayList<>();
```

Comment:
Please revert this. It is inefficient to create an empty list when capacity is known.